packages/theming/apple-theme/src/__tests__/apple-theme.test.ts

```diff
@@ -1,46 +0,0 @@
-import { createAppleTheme } from '../createAppleTheme';
```

Comment:
Could you put the apple-theme tests back please/undo your revert of Nabil's changes?

Reply:
Yup, working on that now. The merge ended up causing issues for some reason